### ISSUE TITLE
Add new stunts

### ISSUE THREAD

**Issue by evangipson** [evangipson/faterator#1]
Should get a list of new ones going for each type:

**Comment by robertpateii** [comment]
yeah this would be good but i would be hesitant to add any more copy directly into my rendering logic (brain.js). you've got the module pattern in place which is good, but not leveraging it at all to actually split out your logic ([see loose augmentation](http://www.adequatelygood.com/JavaScript-Module-Pattern-In-Depth.html)

**Comment by evangipson** [comment]
Done! See [the new data file](https://github.com/evangipson/faterator/blob/master/js/data.js).
